main.cc

```diff
@@ -17,30 +17,26 @@ int main(int argc, char* argv[]) {
             std::cerr << "Usage: webserver <config_file>" << std::endl;
             return 1;
         }
+	
+        // Parse the config file into statements and tokens
+        NginxServerConfigParser server_parser(argv[1]);
 
-        // Get the port number from the config file
-        int port = port_number(argv[1]);
+        // Get the server settings from the config file (i.e. port)
+        server_config server_settings;
+        int port = server_parser.ParseServerSettings(server_settings);
```

Comment:
Just realized that the port that is passed into the server is a short, so maybe change this from an int to a short